servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -504,6 +505,10 @@ public void updatePinList(Set<Long> pinnedInodes) {
    * @throws IOException if an I/O error occurs
    */
   public FileInfo getFileInfo(long fileId) throws IOException {
-    return mFileSystemMasterClient.getFileInfo(fileId);
+    try {
+      return mFileSystemMasterClient.getFileInfo(fileId);
+    } catch (TachyonException e) {
+      throw new IOException(e);
```

Comment:
I feel like there is no need to log the error here since this method won't change the worker's state, and the exception is not caused by worker, it's caused by RPC to master.